eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml

```diff
@@ -77,7 +77,8 @@
   <suppress checks="FinalClass" files="Util.java"/>
 
   <!-- Public API with Impl in Public API TODO confirm, remove -->
-  <suppress checks="NoImplInPublicAPI" files="OpenTelemetryTracer.java"/>
+  <suppress checks="com.azure.tools.checkstyle.checks.NoImplInPublicAPI" files="com.azure.core.tracing.opencensus.OpenCensusTracer.java"/>
```

Comment:
Do we have an issue created for this? Should link to it.

Reply:
This was the issue created for this https://github.com/azure/azure-sdk-for-java/issues/5132

Reply:
Can you add a link to this issue in the checkstyle-suppressions.xml file? It'll be easier for someone to track when they open this file and try to reason about it without knowledge of this PR.

Reply:
added